src/System.CommandLine/Argument.cs

```diff
@@ -122,7 +122,7 @@ public List<Func<CompletionContext, IEnumerable<CompletionItem>>> CompletionSour
         /// <summary>
         /// Gets the default value for the argument.
         /// </summary>
-        /// <returns>Returns the default value for the argument, if defined. Null otherwise.</returns>
+        /// <returns>The default value for the argument, if defined; otherwise, <see langword="null /">.</returns>
```

Comment:
This showed up in checks.  The slash is at the wrong side of the closing quotation mark.